### PR TITLE
Add Squishy65 keyboard

### PR DESCRIPTION
**Pull request by ramonimbao**
## Description

Add the Squishy65 keyboard.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [X] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [X] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [X] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [X] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/ramonimbao/squishy65/squishy65.c [hunk 1]
```diff
@@ -0,0 +1 @@
+#include "squishy65.h"
```
Comment (drashna) [inline keyboards/ramonimbao/squishy65/squishy65.c:1]: Could you add a license header here?

File: keyboards/ramonimbao/squishy65/squishy65.h [hunk 2]
```diff
@@ -0,0 +1,57 @@
+#pragma once
```
Comment (drashna) [inline keyboards/ramonimbao/squishy65/squishy65.h:1]: Could you add a license header, here?

**Review by drashna — CHANGES_REQUESTED** [review]

**Comment by ramonimbao** [comment]
Thanks! I've added it. I just copied it from my `wete` folder and renamed files, so it's missing in my `wete.c` and `wete.h`. Guess I should add a license to those too? Is it okay if I add it from this PR?

**Comment by drashna** [comment]
No, that's fine.  The license header is something that we've recently started enforcing. 

And we have a link for the stuff we're trying to enforce, in general, actually.
https://docs.qmk.fm/#/pr_checklist


If you want to, you can open a separate PR for that keyboard.  (one keyboard per PR makes things easier for us to review, and for the CI to check)

**Review by drashna — APPROVED** [review]

File: keyboards/ramonimbao/squishy65/config.h [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+#pragma once
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xB16B
+#define PRODUCT_ID      0x10B5
+#define DEVICE_VER      0x0065
+#define MANUFACTURER Ramon Imbao
+#define PRODUCT Squishy65
+#define DESCRIPTION 65% keyboard
```
Comment (fauxpark) [inline keyboards/ramonimbao/squishy65/config.h:26]:
```suggestion
```

File: keyboards/ramonimbao/squishy65/keymaps/via/rules.mk [hunk 4]
```diff
@@ -0,0 +1,2 @@
+VIA_ENABLE = yes
+BOOTMAGIC_ENABLE = lite
```
Comment (fauxpark) [inline keyboards/ramonimbao/squishy65/keymaps/via/rules.mk:2]:
```suggestion
```

File: keyboards/ramonimbao/squishy65/rules.mk [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+#BOOTMAGIC_ENABLE = yes	# Virtual DIP switch configuration
+MOUSEKEY_ENABLE = yes	# Mouse keys
+EXTRAKEY_ENABLE = yes	# Audio control and System control
+CONSOLE_ENABLE = yes	# Console for debug
+COMMAND_ENABLE = yes    # Commands for debug and configuration
+SLEEP_LED_ENABLE = yes  # Breathing sleep LED during USB suspend
+NKRO_ENABLE = yes	    # USB Nkey Rollover
+CUSTOM_MATRIX = no # Custom matrix file
+BACKLIGHT_ENABLE = no
+RGBLIGHT_ENABLE = yes
```
Comment (fauxpark) [inline keyboards/ramonimbao/squishy65/rules.mk:17]:
```suggestion
#   change yes to no to disable
#
BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration
MOUSEKEY_ENABLE = yes       # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = yes        # Console for debug
COMMAND_ENABLE = yes        # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = yes           # USB Nkey Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
BLUETOOTH_ENABLE = no       # Enable Bluetooth
AUDIO_ENABLE = no           # Audio output
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Pull request merged**